hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHome.java

```diff
@@ -140,47 +103,21 @@ public String convertToLetter(int size) {
     }
 
     public PhoneHomeParameterCreator createParameters() {
-        ClusterServiceImpl clusterService = hazelcastNode.getClusterService();
-        int clusterSize = clusterService.getMembers().size();
-        Long clusterUpTime = clusterService.getClusterClock().getClusterUpTime();
-        RuntimeMXBean runtimeMxBean = ManagementFactory.getRuntimeMXBean();
-        JetBuildInfo jetBuildInfo = hazelcastNode.getBuildInfo().getJetBuildInfo();
-
-        PhoneHomeParameterCreator parameterCreator = new PhoneHomeParameterCreator()
-                .addParam("version", buildInfo.getVersion())
-                .addParam("m", hazelcastNode.getThisUuid().toString())
-                .addParam("p", getDownloadId())
-                .addParam("c", clusterService.getClusterId().toString())
-                .addParam("crsz", convertToLetter(clusterSize))
-                .addParam("cssz", convertToLetter(hazelcastNode.clientEngine.getClientEndpointCount()))
-                .addParam("cuptm", Long.toString(clusterUpTime))
-                .addParam("nuptm", Long.toString(runtimeMxBean.getUptime()))
-                .addParam("jvmn", runtimeMxBean.getVmName())
-                .addParam("jvmv", System.getProperty("java.version"))
-                .addParam("jetv", jetBuildInfo == null ? "" : jetBuildInfo.getVersion());
-        addClientInfo(parameterCreator);
-        addOSInfo(parameterCreator);
+
+        PhoneHomeParameterCreator parameterCreator = new PhoneHomeParameterCreator();
+        List<MetricsCollector> metricsCollectorList = new ArrayList<>();
+        metricsCollectorList.add(new BuildInfoCollector());
+        metricsCollectorList.add(new ClusterInfoCollector());
+        metricsCollectorList.add(new ClientInfoCollector());
+        metricsCollectorList.add(new MapInfoCollector());
+        metricsCollectorList.add(new OSInfoCollector());
```

Comment:
nit: it would be cleaner to create this list as a field. You can use `java.util.Arrays.asList(...)` to create the array, instead of adding the values to an ArrayList.